### ISSUE TITLE
Dimension White/Black List [Suggestion]

### ISSUE THREAD

**Issue by SureinDragon** [gigaherz/EyesInTheDarkness#9]
First off, thank you for the cool and spooky mod!

Unfortunately when I tested it out in the end, the end was 80% eyes, and there were dozens of them!
I don't really mind them in the end, but I would hope they would be way more rare!

Just wanted to report my findings. Thanks again!

**Comment by gigaherz** [comment]
What you are experiencing is a combination of 2 factors.
First, the mod increases the spawn weight of the Eyes entity the closer it is to halloween, which means eyes are picked for spawning more often than usual.
And second, the way vanilla spawning works. Because the only other entity in the end is normally the enderman, and mc ALWAYS keeps spawning more and more entities every tick until it reaches the mob cap, there will always be an overflow of eyes in the end. You can use this to remove spawns in the end:
```
blacklist = [ "minecraft:void", "minecraft:the_end", "minecraft:small_end_islands", "minecraft:end_midlands", "minecraft:end_highlands", "minecraft:end_barrens" ]
```
Or or try to find a spawn tweaking mod that can adjust the spawn weight per biome / biome category.

**Comment by SureinDragon** [comment]
Aww, thats a pretty neat feature! The higher spawn rate closer to Halloween.

Is that blacklist box you gave for the config file, an in game command, or something else?

I've seen that InControl can help with mob spawning but I've never used it, I will take a look at it.

**Comment by gigaherz** [comment]
Oh that's for the config file, which is stored per-save in the serverconfigs folder of the save (or the world folder as the dedicated server calls it)

**Comment by yvonnemon** [comment]
Hi, I've been trying to implement the blacklist to ban the mob from the end to make my enderman farm more effective, but it doesn't work. It is my own world, as im trying to make it work and then put it on my server. This is how it looks the config file (I've added it from the config menu in the game, not changing the file):
` S:BiomeBlacklist <
        minecraft:the_end
        minecraft:void
        minecraft:small_end_islands
        minecraft:end_midlands
        minecraft:end_highlands
        minecraft:end_barrens
     >`

**Comment by gigaherz** [comment]
1.12 didn't have all those biomes. it was all `minecraft:sky` for the end.

**Comment by yvonnemon** [comment]
Thanks for the fast response, but it didn't work :(
They keep spawning in the end as always...

**Comment by gigaherz** [comment]
I just released v1.0.0 for 1.16.5, which includes a custom spawning system, which among other things, lets you choose which dimensions to allow spawning in. I know your use case was 1.12, but I'm not really interested in backporting, so I'll consider this issue resolved.
